recipes/omniscidb-cuda/recipe/No-boost-cmake-4.8.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 27d2eb8f..05e1aa03 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -75,6 +75,8 @@ else()
+   add_definitions("-DBOOST_LOG_DYN_LINK")
+ endif()
+ 
++set(Boost_NO_BOOST_CMAKE 1)
```

Comment:
Instead of patching like this, you can do this in the cmake command line.

Reply:
Thanks for the hint!
In fact, `No-boost-cmake-4.8.patch` no longer needed and I deleted it.